sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,20 +1,98 @@
 # Release History
 
-## 2.0.0-beta.7 (Unreleased)
+## 2.0.0 (2021-10-12)
+
+We're proud to announce the first GA release of the `@azure/identity` v2 package. The v2 package includes the best parts of v1, plus several improvements.
+
+This changelog entry showcases the changes that have been made. For a v1-to-v2 migration guide, see the [migration guide](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/migration-v1-v2.md).
 
 ### Features Added
 
+#### Plugin API
+
+Identity v2 provides a top-level `useIdentityPlugin` method, which allows using two new plugin packages:
+
+- [@azure/identity-vscode](https://www.npmjs.com/package/@azure/identity-vscode), which provides the dependencies of `VisualStudioCodeCredential` and enables it.
+- [@azure/identity-cache-persistence](https://www.npmjs.com/package/@azure/identity-cache-persistence), which provides persistent token caching.
+- If the `@azure/identity-vscode` plugin isn't used through the `useIdentityPlugin` function, the `VisualStudioCodeCredential` exposed by Identity v2 will throw a `CredentialUnavailableError`.
+
+Most credentials on Identity v2 now include a constructor option called `tokenCachePersistenceOptions`. If set to `tokenCachePersistenceOptions: { enabled: true }`, it will enable persistent token caching on that credential instance.
+
+#### New credentials
+
+Identity v2 includes three new credential types:
+
+- `AzurePowerShellCredential`, which re-uses any account previously authenticated with the `Az.Account` PowerShell module.
+- `ApplicationCredential`, which is a simplified `DefaultAzureCredential` that only includes `EnvironmentCredential` and `ManagedIdentityCredential`.
+- `OnBehalfOfCredential`, which enables the [On-Behalf-Of authentication flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow).
+
+#### Changes to all credentials
+
+Identity v2 enables:
+
+- Support for claims challenges resulting from [Continuous Access Enforcement (CAE)](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-continuous-access-evaluation) and [Conditional Access authentication context](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/granular-conditional-access-for-sensitive-data-and-actions/ba-p/1751775). To disable this behavior, set the environment variable `AZURE_IDENTITY_DISABLE_CP1` to any value. For more about claims challenges, see [Claims challenges, claims requests, and client capabilities](https://docs.microsoft.com/azure/active-directory/develop/claims-challenge).
+- Multi-tenant authentication on all credentials except `ManagedIdentityCredential`. To disable it, set the environment variable `AZURE_IDENTITY_DISABLE_MULTITENANTAUTH`. For more about multitenancy, see [Identity management in multitenant apps](https://docs.microsoft.com/azure/architecture/multitenant-identity/).
```

Comment:
What is the scenario where one would want to disable multi tenant auth?

Reply:
I know we need to support it, and feature-wise it makes sense to me to allow devs to stop this “automated tenant switch” on getToken, but I need to ask what are the concrete scenarios 🤔

Reply:
I have reached out to the Identity crew and the environment variable is to ensure this behavioral change can be reverted. A way to future-proof it.

Reply:
As a user, I would like to know when should I be disabling this feature. Is there anything we can add for that?

Reply:
It really depends on how they’re implementing this. Using our SDKs, they currently will not be encountering scenarios where the getToken request has a `tenantId`. However, if they want multi-tenancy, they will have to implement this on their own (send the new tenantId on getToken). Over time we will be adding more support for multi-tenancy.

I can say something like that. I’ll update this.

Reply:
Let me know how it looks now!